config/lint/lint.xml

```diff
@@ -79,7 +79,7 @@
     <issue id="IconDensities" severity="warning" /> <!--    fluxc-->
     <!-- INTEROPERABILITY -->
     <!-- TODO: https://github.com/wordpress-mobile/WordPress-Android/issues/21065 -->
-    <issue id="UsingMaterialAndMaterial3Libraries" severity="warning" />
+    <issue id="UsingMaterialAndMaterial3Libraries" severity="error" />
```

Comment:
👋 @nbradbury and awesome job with all this! ❤️ 

About switching from `warning` to `error`, just because this [UsingMaterialAndMaterial3Libraries](https://googlesamples.github.io/android-custom-lint-rules/checks/UsingMaterialAndMaterial3Libraries.md.html) check is of `Warning` severity by default, but we are anyway treating all warning as errors ([WordPress/build.gradle#L317](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/build.gradle#L317)), I would recommend deleting this whole configuration and its comment (#21065), doing that we will end-up with the same result.